internal/ctrfactory/ctrfactory.go

```diff
@@ -553,9 +558,35 @@ func (c *container) SpecSetProcessArgs(imageOCIConfig *v1.Image) error {
 	return nil
 }
 
+// SpecAddNamespaces adds the specified namespace paths
```

Comment:
Looks like there's some mishap -- this should not belong to this (renaming) commit.